### DISCUSSION TITLE
diary/2024/08/21/product-analyst-roadmap/

### DISCUSSION THREAD

**Discussion by giscus** [zzsza/blog-comments#155] [Comments]
# diary/2024/08/21/product-analyst-roadmap/

 와 올해 블로그에 쓴 글이 0개였다니.. 충격적이네요. 의식하면서 글을 써보도록 할게요 이번 글은 인프런에 프로덕트 분석가 로드맵을 작성한 내용에 대해 공유해볼게요 강의 만드는 과정의 노하우 공유는 예전에 작성한 인프런에서 강의 만들기 글에 있어요 

https://zzsza.github.io/diary/2024/08/21/product-analyst-roadmap/

[a comment by an automated account is not shown]

**Reply by zzsza** [reply]
안녕하세요!
제가 작성하는 여러 내용들을 잘 봐주셔서 감사합니다..! 
앞으로도 도움이 되는 자료 만들어볼게요 :)

**Comment by Kyeong6** [comment]
안녕하세요 현재 대학교 4학년으로 재학 중이며 데이터 엔지니어를 꿈꾸고 있습니다 학생입니다!
카일님 덕분에 많은 인사이트를 얻고 있습니다. 유튜브 커뮤니티에 BigQuery 기반 데이터 엔지니어링 강의를 올해 말 쯤에 업로드한다고 하셨는데 기대하고 있습니다!

**Reply by zzsza** [reply]
안녕하세요! 
데이터 엔지니어링 강의 준비를 조금씩 하고 있는데, 점진적으로 쪼개서 공유해볼게요. 
관심 가져주셔서 감사합니다!
